### DISCUSSION TITLE
using zustand with reactflow pro examples (groups, nested nodes, dynamic grouping, etc.)

### DISCUSSION THREAD

**Discussion by chandu-intelops** [xyflow/xyflow#4408] [Q&A]
Hi, 
I purchased reactflow pro examples to use for our project. When we started the project we used zustand for state management for basic reactflow components. We want to continue using zustand for reactflow pro components as well, but couldn't find guidelines for that purpose. Could you share any guiding blogs or documentation which shows how to use reactflow pro components with zustand instead of reactflow's internal state manager?

**Comment by pavan-intelops** [comment]
@moklick @peterkogo 
can you please reply?

**Reply by moklick** [reply]
Hey @chandu-intelops @pavan-intelops please reach out via mail for everything that is related to React Flow Pro. 

Any of the pro examples work with Zustand too. If you follow the [state management guide](https://reactflow.dev/learn/advanced-use/state-management), you have everything you need to receive and update nodes. However, please write us an email and we will figure it out.
